.github/workflows/commit-queue.yml

```
@@ -78,4 +78,6 @@ jobs:
           ncu-config set owner "${OWNER}"
 
       - name: Start the commit queue
-        run: ./tools/actions/commit-queue.sh ${OWNER} ${REPOSITORY} ${{ secrets.GITHUB_TOKEN }} $(echo '${{ steps.get_mergable_pull_requests.outputs.data }}' | jq '.repository.pullRequests.nodes | map(.number) | .[]')
+        run: ./tools/actions/commit-queue.sh ${OWNER} ${REPOSITORY} $(echo '${{ steps.get_mergable_pull_requests.outputs.data }}' | jq '.repository.pullRequests.nodes | map(.number) | .[]')
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
          GITHUB_TOKEN: ${{ secrets.GH_USER_TOKEN }}
```

@aduh95 why not this ?

---

We could, the comments would be shown as from the `nodejs-github-bot` instead of `github-actions`, not sure if it matters.

---

It would be consistent with the merge action.

---

I think we should go with your suggestion: it's way simpler and it's probably good for our use case.